src/common/ceph_context.cc

```diff
@@ -425,7 +426,45 @@ void CephContext::do_command(std::string command, cmdmap_t& cmdmap,
         f->dump_string("option", *p);
       }
       f->close_section(); // unknown
-    } else if (command == "log flush") {
+    }
+    else if (command == "config diff get") {
+      std::string ceph_setting;
+      if (!cmd_getval(this, cmdmap, "var", ceph_setting)) {
+        f->dump_string("error", "syntax error: 'config diff get <var>'");
+      } //-- if (!cmd_getval(this, cmdmap, "var", ceph_setting))
```

Comment:
I understand that, but if we have nested blocks, it would look better/more organized. If not *really allowed*, I will take them off.